assets/docs/steinding/steinding.md

```diff
@@ -0,0 +1,12 @@
+# Publisher steinding
+Stein Ding
+
+[![Icon URL]](https://drive.google.com/file/d/160VWvcHH6reFYSt6OM9l_-JjQB8HvWUu/view?usp=sharing)
```

Comment:
It seems like this is not public. Could you double-check?

Reply:
Hi WGierke, sorry for the mistake. Now Everyone can check this photo.